
Thread 3:
Conflicting load by thread 3 at 0x........ size 4
   at 0x........: th (hg05_race2.c:17)
   by 0x........: vgDrd_thread_wrapper (drd_pthread_intercepts.c:?)
Location 0x........ is 0 bytes inside foo.poot[5].plop[11],
declared at hg05_race2.c:24, in frame #? of thread 1
Other segment start (thread 2)
   (thread finished, call stack no longer available)
Other segment end (thread 2)
   (thread finished, call stack no longer available)

Conflicting store by thread 3 at 0x........ size 4
   at 0x........: th (hg05_race2.c:17)
   by 0x........: vgDrd_thread_wrapper (drd_pthread_intercepts.c:?)
Location 0x........ is 0 bytes inside foo.poot[5].plop[11],
declared at hg05_race2.c:24, in frame #? of thread 1
Other segment start (thread 2)
   (thread finished, call stack no longer available)
Other segment end (thread 2)
   (thread finished, call stack no longer available)


ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
